src/main/resources/application.properties

```diff
@@ -115,26 +115,26 @@ finrem.notification.consentedHearing=/list-for-hearing
 finrem.default.assignedToJudgeDefault=${ASSIGNED_TO_JUDGE_DEFAULT_EMAIL:new_application@mailinator.com}
 
 # Document
-document.miniFormTemplate=FL-FRM-APP-ENG-00002.docx
+document.miniFormTemplate=FL-FRM-APP-ENG-00002-DEV.docx
```

Comment:
Make sure you don't check these properties in. They need to be changed back to the actual template names once they have been tested